doc/download.rst

```diff
@@ -65,16 +51,7 @@ If you run **Windows**, this is the option for you OR if you just want to run NE
 After downloading the virtual machine, check out the :doc:`install instructions for Live Media <installation/livemedia>`.
 
 
-* Download the |link-preova|\ |version|\ |link-postova|
-
-.. |link-preova| raw:: html
-
-    <a href="https://nest-simulator.org/downloads/gplreleases/lubuntu-18.04_nest-
-
-.. |link-postova|  raw:: html
-
-     .ova">latest release of the NEST VM image</a>
-
+* Download the `latest release of the NEST VM image <https://nest-simulator.org/downloads/gplreleases/NEST_2.20.0_Virtual_Box_lubuntu_18.04.ova>`_
```

Comment:
See above.

Reply:
I restored the code.
This requires that as soon as there is a new release, a VM exists for the link to work